csrc/scheduler/compile_time_info.h

```diff
@@ -219,26 +219,16 @@ class CompileTimeInfoBase : public PolymorphicBase {
 
 } // namespace HeuristicCompileTime
 
-// Note: Do NOT export this class. MSVC issue with exported class that contains
-// std::vector<unique_ptr<xxx>>: https://godbolt.org/z/3E4e8T1P1
-//! Compile-time information cache for `canSchedule` and
-//!  `getHeuristics` interfaces. Each cache instance
-//!  stores information that could be inferred at compile
-//!  time in a fusion and therefore corresponds to an
-//!   instance of FusionExecutor.
-//!  Since each instance of FusionExecutor has a unique
-//!   heuristic type, this cache also has a heuristic
-//!   type to simplify data validation.
-//!  HeuristicSummary has two modes of operation:
-//!  - when in `recording` mode, the information is not available
-//!     in the cache and entries can be written and stored.
-//!  - when not in `recording` mode, compiled-time data has
-//!     been stored in this cache and the entries can be accessed
-//!!    but new entries can no longer be inserted.
+//! Note: Do NOT export this class. MSVC issue with exported class that contains
+//! std::vector<unique_ptr<xxx>>: https://godbolt.org/z/3E4e8T1P1
+//! Compile-time information cache for `canSchedule` and `getHeuristics`
```

Comment:
nit: Not directly related to the changes in this PR, but I always find this name a little confusing. It's a caching system for some fusion properties that are used to generate heuristics. "HeuristicSummary" seems to indicate it's a summary of heuristics.